tests/Fixtures/functions/enum_cases/valid.test

```diff
@@ -3,13 +3,13 @@
 --CONDITION--
 \PHP_VERSION_ID >= 80100
 --TEMPLATE--
-{% for c in enum_cases('Twig\\Tests\\DummyBackedEnum') %}
+{% for c in enum_cases('Twig\Tests\DummyBackedEnum') %}
```

Comment:
Can we double the tests to also prove that double-escaping still work (for all 3 cases maybe)?

Reply:
Done